src/isOS.js

```diff
@@ -0,0 +1,13 @@
+export const isWindows = () => {
```

Comment:
Reading through this PR, I'm not sure that these checks are good enough.
Back to the drawing board for these.